src/common/snippets/src/utils/loop_utils.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/utils/loop_utils.hpp"
+
+#include "snippets/lowered/loop_port.hpp"
+#include "snippets/op/buffer.hpp"
+#include "snippets/op/memory_access.hpp"
```

Comment:
Minor: Looks like these includes are redundant. Remove please them if it's so

Reply:
Removed, thanks